code-challenges.test.js

```diff
@@ -49,12 +123,56 @@ const studyMinutesWeek2 = {
 }
 // Expected output: [10, 15, 20, 45, 60, 65, 100]
 
+// describe("minutes", () => { 
+//   it("takes in an object and returns an array of the values sorted from least to greatest", () => {
+//     expect(minutes(Object.values(studyMinutesWeek1))).toEqual([15, 15, 20, 30, 30, 60, 90])
```

Comment:
Try to avoid adding any logic into your tests.  We can use Object.values inside our function to make this more consistently dynamic.